specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-10-15-preview/EventGrid.json

```diff
@@ -5501,12 +5654,12 @@
           "readOnly": true
         },
         "name": {
-          "description": "Name of the resource",
+          "description": "Name of the resource.",
           "type": "string",
           "readOnly": true
         },
         "type": {
-          "description": "Type of the resource",
+          "description": "Type of the resource.",
           "type": "string",
           "readOnly": true
         }
```

Comment:
ARM requires a new top level property `systemData` for all tracked / proxy resources.
https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc027-systemdata-support

You can reference the one in common-type.
https://github.com/Azure/azure-rest-api-specs/blob/a16488bbd0d17551c729e22ea6ac6f7c6588a0a3/specification/common-types/resource-management/v2/types.json#L445

Reply:
@j5lim  This seems like a new requirement and requires additional work and testing. It is outside the scope of this change as I am not even adding a new resource type. 
I will create a work item for us to investigate this new property and add that as a separate PR.